libdirections-offboard/src/test/java/com/mapbox/navigation/route/offboard/router/NavigationRouteTest.kt

```diff
@@ -129,10 +145,18 @@ class NavigationRouteTest {
     @Test
     fun addWaypointTargetsIncludedInRequest() {
         val navigationRoute = NavigationRoute.builder(context)
-            .accessToken(ACESS_TOKEN)
-            .origin(Point.fromLngLat(1.0, 2.0))
-            .destination(Point.fromLngLat(1.0, 5.0))
-            .addWaypointTargets(null, Point.fromLngLat(0.99, 4.99))
+            .routeOptions(
+                RouteOptionsNavigation.builder().accessToken(ACESS_TOKEN)
+                    .origin(origin)
+                    .destination(destination)
+                    .waypointTargets(
+                        arrayOf(
+                            Point.fromLngLat(0.99, 4.99),
+                            Point.fromLngLat(1.99, 5.99)
+                        )
+                            .joinToString(separator = ";") { "${it?.latitude()},${it?.longitude()}" }
```

Comment:
NIT no new line needed

```suggestion
).joinToString(separator = ";") { "${it?.latitude()},${it?.longitude()}" }
```